src/frontend/src/test-e2e/selenium.test.ts

```diff
@@ -291,8 +291,10 @@ test("Screenshots", async () => {
       await welcomeBackView.waitForDisplay();
       await welcomeBackView.fixup();
       await screenshot("08-new-device-login", browser);
+      await browser.pause(3000);
```

Comment:
The flaky issue occurs, because we click to early. So it's no use to wait longer in " await singleDeviceWarningView.waitForDisplay();", we have to wait a bit before clicking the link.

Reply:
In my opinion pauses in tests are rarely a solution. Why do we click too early? Does that mean the user may also click too early?